src/App.js

```diff
@@ -1,13 +1,21 @@
+
 import "./App.css";
+import { Login } from "./pages/authentication/login/login";
+import { SignUp } from "./pages/authentication/signUp/signUp";
+import { History } from "./pages/history/history";
 import { Home } from "./pages/home/home";
 import { Playlist } from "./pages/playlist/playlist";
-
+import { WatchLater } from "./pages/watchLater/watchLater";
```

Comment:
You can create a index file and exports pages from there.

Reply:
ok, thank you